root/etc/cont-init.d/60-striptracks-init

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/with-contenv bash
```

Comment:
this init script probably should be named `98-striptracks` for uniformity with all other mods unless there is a specific reason to run this earlier

Reply:
Fixed